src/js/view/controls/controls.js

```diff
@@ -19,6 +19,8 @@ import ShortcutsTooltip from 'view/controls/shortcuts-tooltip';
 import FloatingCloseButton from 'view/floating-close-button';
 
 require('css/controls.less');
+// Allows preventScrolling focus option on IE/Safari.
+require('../../../../node_modules/focus-options-polyfill');
```

Comment:
we should put an import into `src/js/polyfills` and import that here for clarity and file structure. that file in `src/js/polyfills` can simply run this require